src/modules/person/api/personenuebersicht/dbiam-personenuebersicht.controller.ts

```diff
@@ -78,7 +78,7 @@ export class DBiamPersonenuebersichtController {
 
         // Find all Personen on child-orgas (+root orgas)
         const scope: PersonScope = new PersonScope()
-            .findBy({ organisationen: organisationIDs })
+            .findBy({ ids: queryParams.personIds, organisationen: organisationIDs })
```

Comment:
Scope in the controler is bad practice, but since it was not the scope of the ticket to fix this, it is acceptable.